modules/labs-react/expandable/lib/Expandable.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import {createContainer, ExtractProps} from '@workday/canvas-kit-react/common';
+
+import {ExpandableContent} from './ExpandableContent';
+import {ExpandableTarget} from './ExpandableTarget';
+import {ExpandableStartIcon} from './ExpandableStartIcon';
+import {ExpandableEndIcon} from './ExpandableEndIcon';
+import {ExpandableTitle} from './ExpandableTitle';
+import {ExpandableAvatar} from './ExpandableAvatar';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {useExpandableModel} from './useExpandableModel';
+
+export const ExpandableModelContext = useExpandableModel.Context;
+
+export type ExpandableModelConfig = Partial<typeof useExpandableModel.defaultConfig> &
+  typeof useExpandableModel.requiredConfig;
+
+export interface ExpandableProps extends ExpandableModelConfig, ExtractProps<typeof Flex, never> {
```

Comment:
I think now you can remove `ExpandableModelConfig` since `createContainer` extracts out the types based on the `modelHook`